app/views/component-listing.html

```diff
@@ -38,6 +38,9 @@
 				<tr><th></th><th></th><th>Updated</th></tr>
 			</thead>
 			<tbody>
+			<tr class="component-group empty-search">
+				<td><h3>Can't find what you are looking for</h3></td>
```

Comment:
At the moment the bottom border doesn't span across the entire row, you'll need to add either 2 more empty `<td>`'s (see line 50) or use colspan to get the one cell to span all 3 columns.